modules/installation-configuration-parameters.adoc

```diff
@@ -656,6 +648,23 @@ ifdef::ibm-power-vs[]
 |The system type must be either `e980` or `s922`.
 endif::ibm-power-vs[]
 |====
+[.small]
+--
+1. Not all CCO modes are supported for all cloud providers. For more information about CCO modes, see the "Managing cloud provider credentials" entry in the _Authentication and authorization_ content.
++
+ifdef::aws,gcp[]
+[NOTE]
```

Comment:
I am calling out that we try to avoid stacked admonitons, but looking at this table, and the surrounding content, I am not sure what to suggest as a better alternative.

Reply:
Yeah, I don't know what the eventual solution to these tables will be, but I suspect this workaround I have will not scale because of that consideration. At least a few people are trying to get a better solution to this situation, so hopefully we'll have something slick soon :crossed_fingers: